fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1091,9 +1091,10 @@ struct AutoQuorumChange : IQuorumChange {
 		vector<Future<Optional<LeaderInfo>>> leaderServers;
 		for( int i = 0; i < coord.clientLeaderServers.size(); i++ )
 			leaderServers.push_back( retryBrokenPromise( coord.clientLeaderServers[i].getLeader, GetLeaderRequest( coord.clusterKey, UID() ), TaskPriority::CoordinationReply ) );
-		Optional<vector<Optional<LeaderInfo>>> results = wait( timeout( getAll(leaderServers), CLIENT_KNOBS->IS_ACCEPTABLE_DELAY ) );
-		if (!results.present()) return false;  // Not all responded
-		for(auto& r : results.get())
+		Optional<vector<Optional<LeaderInfo>>> results =
+		    wait(timeout(getAll(leaderServers), CLIENT_KNOBS->IS_ACCEPTABLE_DELAY));
+		if (!results.present()) return false; // Not all responded
```

Comment:
since you are changing these lines, it would be great if you could add `{}` for the one-line statement.